recipe/conda_build_config.yaml

```diff
@@ -1,2 +1,5 @@
 openjdk:
   - 8
```

Comment:
If pyjnius works with Java 9 and you add that here, then you will get twice as many builds and would help verify that what I think you want to do is actually working.

Reply:
I know but at the moment `pyjnius` is not compatible with Java >=9. We need https://github.com/kivy/pyjnius/issues/304 to be fixed first.